src/API.ts

```diff
@@ -42768,6 +42768,17 @@ export type GetNotesQueryVariables = {
   id: string,
 };
 
+export type GetCustomNotes = {
```

Comment:
I'm ~99% sure that this will get overwritten by GraphQL codegen since the file is auto generated.

Maybe add the type definition in [`types.ts`](https://github.com/themeetinghouse/web/blob/master/src/components/types.ts) or just in `Notes.tsx` since it won't be used anywhere else.

Reply:
Not sure if I've done it correctly but it seems to be working. Let me know if there are any more changes that need to be made.

Reply:
You don't appear to be using the type definition in `types.ts` since you also added it to `Notes.tsx` and reference it within that file locally. Everything will work, but I would remove the definition from `types.ts` since it isn't doing anything.